java/src/org/openqa/selenium/remote/http/ClientConfig.java

```diff
@@ -48,19 +49,28 @@ protected ClientConfig(
       Filter filters,
       Proxy proxy,
       Credentials credentials,
-      SSLContext sslContext) {
+      SSLContext sslContext,
+      HttpClient.Version version) {
```

Comment:
Wouldn't this couple the `ClientConfig` with a particular client implementation? In this case, the current Java 11 HTTP implementation. 

Is `HttpClient.Version` accepted in other Java HTTP clients?

Reply:
You're right, this should be a String. No idea other HTTP clients. If they want it, they can use it